learning-another-language/hello-world.md

```diff
@@ -1,57 +1,47 @@
 # Hello, World!
 
-<iframe src="https://adaacademy.hosted.panopto.com/Panopto/Pages/Embed.aspx?id=1273d731-b286-4a6e-9b0c-ad4000f5c3d3&autoplay=false&offerviewer=true&showtitle=true&showbrand=false&start=0&interactivity=all" height="405" width="720" style="border: 1px solid #464646;" allowfullscreen allow="autoplay"></iframe>
+<!-- REPLIT-REMOVAL-UPDATE  -->
+<!-- <iframe src="https://adaacademy.hosted.panopto.com/Panopto/Pages/Embed.aspx?id=1273d731-b286-4a6e-9b0c-ad4000f5c3d3&autoplay=false&offerviewer=true&showtitle=true&showbrand=false&start=0&interactivity=all" height="405" width="720" style="border: 1px solid #464646;" allowfullscreen allow="autoplay"></iframe> -->
 
 ## Introduction
 
 A well-known exercise to accomplish when learning another programming language is "Hello, World!" We'll know we're successful if we see the text `Hello, World!` appear somewhere after running the code.
 
-## Running Code in the REPL
+![A picture of Grumpy Cat with the text "Hello World" over it](../assets/learning-a-new-language_hello-world_grumpy-cat-hello-world.jpg)  
+_Fig. Grumpy Cat sharing a "Hello World"_
+
+## Running the Example Code
 
 To run JavaScript code, we need to consider its _runtime environment_. A **runtime environment** is a set of technologies that determine _how_ to execute the code.
 
 Some examples of runtime environments that can run JavaScript:
 
 - The runtime environment of a web browser like Mozilla Firefox and Google Chrome
 - [Node.js](https://nodejs.org/en/)
-- REPLs like [replit.com](https://replit.com/)
-
-For this portion of the curriculum, we will use [replit.com](https://replit.com/).
-
-The following steps will let us create and run JavaScript programs with ease:
-
-1. Create an account on [replit.com](https://replit.com/) and log-in
-1. Visit [the Hello, World! REPL](https://replit.com/@adacore/hello-world#index.js)
-
-When we open a REPL that we don't own, we will see a page with the project name, and a play button to run the project. We can toggle the view to show the code. We can also make a copy of the project in our own account by clicking Fork.
-
-![JavaScript readonly REPL. The browser displays the name of the project, hello-world, and a large green button to run the project. A toggle labelled Code and button labelled Fork are also visible.](../assets/learning-a-new-language_hello-world_repl-unowned.png)
+- Online REPLs like [replit.com](https://replit.com/)
 
-Let's Fork this project and see what that lets us do!
+Our VS Code environment should be set up from a previous lesson, so we'll continue to practice writing and running code locally to get used to that development flow. 
 
-Now, on the right side, we see the _console_. We can use this console to execute small, quick JavaScript snippets, such as `2 + 3`, `4 * 5`, or any other valid code.
+To kick off our JavaScript journey, we'll take a look at [this example repo containing a Hello World application](https://github.com/AdaGold/hello-world-js).
```

Comment:
We link to the same repo twice within a sentence of each other. Wasn't sure if we needed to do that just in case students think they are two separate repos. 

```suggestion
To kick off our JavaScript journey, we'll take a look at an example repo containing a Hello World application.
```

Reply:
I had a similar feeling, but was waffling on changing it, I'm good removing the link here.